src/components/ItemsList/ItemsList.js

```diff
@@ -0,0 +1,40 @@
+const repeat = (num, value, sep) => {
+	let res = '';
+	for (let i = 0; i < num; i++) {
+		res += value;
+		if (i !== num - 1) res += sep;
+	}
+	return res;
+}
+
+export const ItemsList = ({
+	Component,
+	itemsMax,
+	maxInRow,
+	items = [],
+	style = {},
+}) => {
+	let filteredByMax = items;
```

Comment:
можно чейнинг оператор использовать, чтобы убрать let и иф
```suggestion
	const filteredByMax = items?.filter((_, index) => index < itemsMax);;
```